app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<section class='login-container'>
+  <div class='login'>
+    <h2>Sign up</h2>
+
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+
+      <div class="field">
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder:'User-email', class: 'email' %>
+      </div>
+
```

Comment:
- [ ] Kindly add input fields of `bio`, `name`, and `photo` in order for the registration process to work fine.